command/server/server.go

```diff
@@ -197,15 +197,15 @@ func setFlags(cmd *cobra.Command) {
 		&params.jsonRPCBatchLengthLimit,
 		jsonRPCBatchRequestLimitFlag,
 		defaultConfig.JSONRPCBatchRequestLimit,
```

Comment:
Just curious. Is it ok that default value is not zero?

Reply:
@zivkovicmilos I see that we approved that on this PR https://github.com/0xPolygon/polygon-edge/pull/638. But I agree, we should consider having default 0, and then if someone wants can adjust it

Reply:
Sounds reasonable, agree lets leave it like it is

Reply:
@Kourin1996 please resolve so we can merge this